src/OmniSharp.Http/OmniSharp.Http.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net46;netstandard1.6</TargetFrameworks>
+        <WarningsAsErrors>true</WarningsAsErrors>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <!-- Needed for Microsoft.Composition -->
+        <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);dotnet5.4;portable-net45+win8</PackageTargetFallback>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="1.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="1.1.0" />
+        <ProjectReference Include="..\OmniSharp.Host\OmniSharp.Host.csproj" />
```

Comment:
This feels like a layering violation. Currently, nothing references `OmniSharp.Host` and it really shouldn't provide public API -- just implementations. Could API be moved into `OmniSharp.Abstractions` instead of referencing `OmniSharp.Host`?

Reply:
Perhaps we should rename `OmniSharp.Host` to `OmniSharp.Hosting`?

Reply:
FWIW, I'd be interested in what you think about a design where there's a single startup project that dynamically loads the correct server based on command-line args. Would that be possible?

Reply:
It's totally doable now since we're only bound to the desktop framework.  That would be mean that we should ship all the hosts in package, which would mean we're bring kestrel with us going forward.  And then the same thing would happen if/when we support the new sockets api.  I don't have a preference, but there would be extra "baggage" as it were.

Reply:
If we design for this case, one could imagine that the "transport hosts" could be plugins.

Reply:
Note: I'm not asking for further changes here. We should get this PR in and start testing and refactoring.